### ISSUE TITLE
Test Case: Build Button Component

### ISSUE THREAD

**Issue by CatQueenCodes** [I3uckwheat/cards-of-carousal#274]
**[QA User Testing protocol Wiki](https://github.com/I3uckwheat/cards-of-carousal/wiki/QA-User-Testing-Protocol)**

**Tests must be conducted using the live site, not your locally run version.** 
**Live Site: https://cardsofcarousal.com/**

Please keep track of the following in the event you find a bug:
- What data you input/the output (if necessary for your test scenario)
- The steps taken during the test
- What browser & device are being used
- Screenshot/video screen capture for the bug ticket.

# Test Scenario:
Based on User Story: #7  

All buttons should have `Inactive` styling by default (white with black text) aside from `LET'S CAROUSE!` button on the Player Join Screen where you type your name and join code in, which should have the `Active` styling (black with white text). When you hover over a button it should have the `Hover` styling (grey with white text). All text should be centered in the button.

## Figma Design
![image](https://user-images.githubusercontent.com/4411701/102017137-e7f4f800-3d19-11eb-8550-f6b4402e6d60.png)

Figma Link with screen names for tests below: https://www.figma.com/file/DhhWwBRIIL6fqcYzayUs99/CAH?node-id=0%3A1

## Completion Criteria
-You must play through the game as host and as a player and as czar to check ALL buttons. To do this: you will need to open 3 tabs or windows: In one you will be the host, in 2 others, you will create players. 

## iOS Safari:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

## iOS Chrome:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text
- [ ] Remove iOS testing tag once all iOS testing is complete

## Android Chrome:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text
- [ ] Remove Android testing tag once all Android testing is complete

## Desktop Safari:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

## Desktop Chrome:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

## Desktop FireFox:
- [ ] Welcome Screen: `JOIN` & `HOST` button have `Inactive` styling with centered text

**Playing as the host:**
- [ ] Host Setup Screen: `START CAROUSING` & `CLOSE GAME` have `Inactive` styling with centered text

**Playing as the player:**
- [ ] Player Join Screen `LET'S CAROUSE` button has `Active` styling with centered text
- [ ] Player Select Screen A `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text

**Playing as the Czar:**
- [ ] Player Select Screen A (czar version) `SUMBIT` & `CLEAR` buttons have `Inactive` styling with centered text
- [ ] Remove Desktop testing tag once all Desktop testing is complete

## Things to consider/try:
- Anything additional you can think of that fits within the scope of this ticket

## Accessibility (Eric)
- [ ] Uses semantic tags
- [ ] Ability to tab through
- [ ] Screen reader compatible
- [ ] Visually passing

# If you find a bug
Link to a QA bug ticket template and instructions ---- Gotta write this up
